src/bootstrap/test.rs

```diff
@@ -708,6 +708,81 @@ impl Step for RustdocUi {
     }
 }
 
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub struct RustdocGUI {
+    pub host: Interned<String>,
+    pub target: Interned<String>,
+    pub compiler: Compiler,
+}
+
+impl Step for RustdocGUI {
+    type Output = ();
+    const DEFAULT: bool = true;
```

Comment:
Please remove this as a default running test suite, we'll only want to run it on that specific CI builder.